.github/workflows/lint.yml

```diff
@@ -0,0 +1,39 @@
+name: lint

```

Comment:
This doesn't give a great title in the status checks
![image](https://github.com/justeattakeaway/JustSaying/assets/2156707/2bab9397-32fa-4678-a325-5d4200c35814)

I think the following would be better, as it would give us `actionlint/lint`
```suggestion
name: actionlint

```

Reply:
I'd left it generic so we could add other linters in the future, but I've changed it to `lint-actions`.